api/src/main/java/org/apache/iceberg/Snapshot.java

```diff
@@ -104,4 +104,11 @@
    * @return the location of the manifest list for this Snapshot
    */
   String manifestListLocation();
+
+  /**
+   * Return this snapshot's sequence number, or 0 if the table has no snapshot yet.
```

Comment:
In which case will this actually return 0? If there is no snapshot, then there is no `Snapshot` object, right?

Reply:
It would make sense to return 0 if the snapshot was written without a sequence number, and to use a primitive, right?